src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
```

Comment:
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/5c20ac0b-e9a0-4573-93c6-cac563e42fc6)

Reply:
not fixed, use name() method

Reply:
Fixed